src/Microsoft.IdentityModel.Protocols.WsFederation/MetadataSerializer.cs

```diff
@@ -27,8 +27,14 @@
 
 namespace Microsoft.IdentityModel.Protocols.WsFederation
 {
+    /// <summary>
+    /// Serialize metedata.
```

Comment:
This is a class description, so it should read something like "This class serializes the metadata to be used ..." so someone reading the docs could tell what its usage is.